src/lib.rs

```diff
@@ -208,13 +200,9 @@ impl HoneycombPipelineBuilder {
             "opentelemetry-honeycomb-rs",
             Some(env!("CARGO_PKG_VERSION")),
         );
-        let provider_guard = opentelemetry::global::set_tracer_provider(provider);
+        let _ = opentelemetry::global::set_tracer_provider(provider);
```

Comment:
this seems like the most impactful API change from the upgrade, introduced in https://github.com/open-telemetry/opentelemetry-rust/pull/444. since there's no longer a provider guard that flushes the remaining traces on `Drop`, it looks like clients need to call `opentelemetry::global::shutdown_tracer_provider()` now. mind updating the examples to include this call?

Reply:
Good point. I've added it.